recipes/xmsgrid/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "3.0.5" %}
+
+package:
+  name: xmsgrid
+  version: {{ version }}
+
+source:
+  url: https://github.com/Aquaveo/xmsgrid/archive/{{ version }}.tar.gz
+  sha256: de0b6a59afb4fb1929b81c07e3078c37e15a0c01ba6999db78b3ed50a4335ae4
+
+build:
+  number: 0
+  skip: True  # [not py36]
+  merge_build_host: true
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.0.0
+    # just to avoid build/host merge issues
+    # but these should be in host.
```

Comment:
```suggestion
  host:
```

Reply:
We tried this with our last library, xmscore, and @ocefpaf and I could not get it to work any other way